docs/source/concoreexample.rst

```diff
@@ -0,0 +1,46 @@
+Concore examples
+================
+
+Introduction
+------------
+This feature makes it easy for users to share their studies, examples, and program files. With this feature, you can contribute your work without having to use GitHub.
+
+
+How to use?
+-----------
+
+- Open **concore editor** by clicking on https://controlcore-project.github.io/concore-editor/
+
+- Click on ``Contribute`` button at top-right of the editor
+
+- To contribute examples, concore fri(server) needs to be switched on. To do so refer https://control-core.readthedocs.io/en/latest/concoreeditor.html#how-to-use
+
+- Fill in the details as follow:-
+    These are the necessary arguments
+        - Study Name - title of the study
+        - Study Path - full directory path where the study is located as shown below.
+        - Author Name - name of the person who created that study
+
+    You can also provide the options arguments by clicking  ``Show Advance Options`` button
+        - Branch Name - the branch name you want 
+        - Title of Study - a short title which will displayed as title of pull request at github
+        - Description of Study - a description which will be displayed as body of the pull request at github
+
+
+``Note: If you are not familiar with Github, we recommend you to skip the optional fields``
+
+.. image:: images/ce-contribute.png
+  :width: 700
+
+- Click on ``Generate PR`` button and wait until it processes your request
+
+.. image:: images/ce-contribute-loading.png
+  :width: 700
+
+- After it gets completed, you can check at the link given in the message
+
+.. image:: images/ce-contribute-success.png
+  :width: 700
+
+
+``Note: Successful submission doesnot mean that your example is added to our repository, It will take time for us to review and accept the example.So, you can mention your email in the description field mentioned above so that if your example got accepted then we will let you know.``
```

Comment:
Rename "example" here with "study"

Don't use "example" in this feature. We/users are not contributing "examples." They are contributing studies.

Reply:
Yeah I will update that, thanks for pointing it out